locallib.php

```diff
@@ -2626,6 +2626,32 @@ function theme_boost_union_is_active_theme() {
     }
 }
 
+/**
+ * Helper function to show an alert if Boost Union is not the active theme but someone tries to access
+ * Boost Union's settings or files.
+ */
+function theme_boost_union_is_not_active_alert() {
+    global $SCRIPT;
+
+    if (theme_boost_union_is_active_theme()) {
+        return;
+    }
+
+    $ispluginpath  = str_starts_with($SCRIPT, '/theme/boost_union');
```

Comment:
As you have placed the theme_boost_union_is_not_active_alert() calls on particular settings pages, it should not be necessary to check paths here again.

Please remove this part and double check again if the notifications just appear where they should appear.